usr/local/sbin/restic_backup.sh

```diff
@@ -71,20 +71,14 @@ wait $!
 restic forget \
 	--verbose \
 	--tag $BACKUP_TAG \
+        --prune
 	--group-by "paths,tags" \
 	--keep-daily $RETENTION_DAYS \
 	--keep-weekly $RETENTION_WEEKS \
 	--keep-monthly $RETENTION_MONTHS \
 	--keep-yearly $RETENTION_YEARS &
 wait $!
 
-# Remove old data not linked anymore.
-# See restic-prune(1) or http://restic.readthedocs.io/en/latest/060_forget.html
-restic prune \
-	--option b2.connections=$B2_CONNECTIONS \
```

Comment:
This should maybe be migrated to restic-forget?

Reply:
I've added the B2 connection limit to restic forget in 429ee40.